esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/workflow.rst

```diff
@@ -11,6 +11,7 @@ The |RTW| performs the following steps:
 ``install_cold``
   :Description:
      Activates the environment for |ESMValTool|, based on the ``SITE`` provided
+     Installs the latest versions of |ESMValTool| and |ESMValCore| from github
```

Comment:
"Checkouts" or "Clones" rather than "Installs" would be better here, as we're not actually installing ESMValCore or ESMValTool:
```suggestion
     Checkouts the latest versions of |ESMValTool| and |ESMValCore| from GitHub
```

Reply:
> "Checkouts" or "Clones" rather than "Installs" would be better here, as we're not actually installing ESMValCore or ESMValTool:

9f790b5199e447d6c43851b0660dd26e43ee83f7